dkg-test-orchestrator/config/test_n3t2.toml

```diff
@@ -0,0 +1,6 @@
+# the server bind addr
+bind = "127.0.0.1:7777"
+n_clients = 3
+threshold = 2
+min_simulated_latency = "100ms"
+positive_cases = 10
```

Comment:
Some docs would be useful here on what each of these options means

Reply:
They are in the comments inside the rust struct representing the TOML, but, I can copy them over for this test config.